Plugin.Maui.Apptentive/Plugin.Maui.Apptentive.csproj

```diff
@@ -52,8 +52,8 @@
 
 	<ItemGroup Condition="'$(TargetFramework)' == 'net8.0-android'">
 	 <PackageReference Include="ApptentiveFeedback" Version="6.8.0" />
-		<!-- For development:
-		<ProjectReference Include="..\ApptentiveKit.Android\ApptentiveCore\ApptentiveCore\ApptentiveCore.csproj" />
+		<!-- For development: -->
+		<!-- <ProjectReference Include="..\ApptentiveKit.Android\ApptentiveCore\ApptentiveCore\ApptentiveCore.csproj" />

```

Comment:
Does this also still need to be commented out?

Reply:
Yes. This is only for development (testing purpose) as per Luqmaan. I will make a note to check with him before release.